config/production.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Remove.

Reply:
Not sure if you can, this is something is required by config module, will confirm shortly anyway

Reply:
Done!